### DISCUSSION TITLE
原生UI

### DISCUSSION THREAD

**Discussion by xfgryujk** [xfgryujk/blivechat#166] [插件]
# 原生UI

提供托盘图标和用来看弹幕的窗口

* 支持置顶窗口和设置不透明度
* 支持付费消息分开显示
* 统计弹幕数、付费用户等信息，可以导出Excel表格

![1](https://github.com/xfgryujk/blivechat/assets/8419596/24c85bff-387b-40d1-a133-759823ebe2fa)
![2](https://github.com/xfgryujk/blivechat/assets/8419596/a7917d45-24f9-43c7-b455-6d778dbc2c52)
![3](https://github.com/xfgryujk/blivechat/assets/8419596/3b67fbed-1a81-43cd-aa01-c9043ab2fb3f)
![4](https://github.com/xfgryujk/blivechat/assets/8419596/95688b95-6a0b-4baa-a412-b3feb3315f0e)
![5](https://github.com/xfgryujk/blivechat/assets/8419596/b3e15646-bd72-450f-8e04-21ff6754b531)

## 下载

* [native-ui-1.0.1-windows-x64.zip](https://github.com/user-attachments/files/26330246/native-ui-1.0.1-windows-x64.zip) SDK v1.0.1
* [native-ui-1.0.0-windows-x64.zip](https://github.com/xfgryujk/blivechat/files/14636646/native-ui-1.0.0-windows-x64.zip) SDK v1.0.0

### v1.0.1 2026-03-29

* 统计数据不计入跨房弹幕
* 升级SDK到v1.0.1

### v1.0.0 2024-03-18

第一个发布版本

**Comment by luchenhao** [comment]
显示没有任何已连接的房间是什么问题？

**Reply by xfgryujk** [reply]
要开启通过服务器转发消息，且打开房间页面

**Comment by Mr54233** [comment]
这个能在悬浮窗口上挂载css吗？

**Reply by xfgryujk** [reply]
你可以修改`config.ini`里的URL参数，让它使用预设CSS
